pkg/pullsync/pullsync.go

```diff
@@ -196,7 +203,8 @@ func (s *Syncer) SyncInterval(ctx context.Context, peer swarm.Address, bin uint8
 		}
 
 		delete(wantChunks, addr.String())
-
+		s.metrics.DbOpsCounter.Inc()
+		s.metrics.DeliveryCounter.Inc()
```

Comment:
shouldn't we increase this only if `s.storage.Put` is successfull?

Reply:
we're counting deliveries (which is a protocol receiving a chunk), not db operation errors